src/header/__tests__/Header.js

```diff
@@ -34,13 +35,15 @@ describe('Header Component', () => {
       </Header>
     );
 
-    expect(component.find('Button').length).toBe(2);
+    expect(component.length).toBe(1);
+    expect(toJson(component)).toMatchSnapshot();
```

Comment:
Should we be snapshotting here ?. Can we test the actually functionality describe by the test case 😉
`should render multiple children when passed in`

`expect(component.children().length).toBe(2)`

or more explicitly 

`expect(component.find('Button').length).toBe(2)`

same with the other cases :)